messages/run-command.md

```diff
@@ -52,45 +52,41 @@ We're continually improving Salesforce Code Analyzer. Tell us what you think! Gi
 
     <%= config.bin %> <%= command.id %> --rule-selector eslint:getter-return --rule-selector no-inner-declarations
 
-# command.examples.temporarily-hidden-for-now
-
-- Specify three method starting points to be applied to the path-based recommended rules of the Salesforce Graph Engine, "sfge", while specifying the "./src" folder as the workspace to be used to build the source graph:
-
-    <%= config.bin %> <%= command.id %> --rule-selector sfge:Recommended --workspace ./src --path-start ./src/classes/Utils.cls#init --path-start ./src/classes/Helpers.cls#method1;method2
-
 # flags.workspace.summary
 
-Set of files you want to include in the code analysis. 
+Set of files that make up your workspace.
 
 # flags.workspace.description
 
-Typically, a workspace is a single project folder that contains all your files. But it can also consist of one or more folders, one or more files, and use glob patterns (wildcards). If you specify this flag multiple times, then your workspace is the sum of the files and folders. 
+Typically, a workspace is a single project folder that contains all your files. But it can also consist of one or more folders, one or more files, and use glob patterns (wildcards). If you specify this flag multiple times, then your workspace is the sum of the files and folders.
+
+Note that even if you wish to only target (via the `--target` flag) a subset of the files within your workspace, some engines may still require your complete workspace to perform its analysis on your targeted files. For example, the Salesforce Graph Engine may need to compile your entire project in order to properly build a graph so that it may perform data flow analysis on the paths that start in your targeted files.
```

Comment:
```suggestion
Some engines often need your entire code base to perform an analysis, even if you want to target only a subset of the files within your workspace , such as with the `--target` flag. For example, the Salesforce Graph Engine might need to compile your entire project in order to properly build a graph so it can perform a data flow analysis on the paths that start in your targeted files.
```